python/whylogs/core/relations.py

```diff
@@ -0,0 +1,193 @@
+import re
+from abc import ABC, abstractmethod
+from enum import Enum
+from typing import Any, Callable, Optional, Union
+
+from whylogs.core.configs import SummaryConfig
+from whylogs.core.metrics.metrics import Metric
+
+
+class ValueGetter(ABC):
+    @abstractmethod
+    def __call__(self) -> Union[str, int, float]:
+        raise NotImplementedError
+
+    @abstractmethod
+    def serialize(self) -> str:
+        raise NotImplementedError
+
+
+class LiteralGetter(ValueGetter):
```

Comment:
can we have a test for LiteralGetter?

Reply:
Almost all the tests exercise `LiteralGetter`.